src/App.tsx

```diff
@@ -1,61 +1,125 @@
-import './App.scss';
+import { useState, ChangeEvent, useEffect } from "react";
+import "./App.scss";
+import { TodoList } from "./components/TodoList";
+import { getUser, tasks } from "./helpers.tsx/helpers";
+import usersFromServer from "./api/users";
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: {
+    id: number;
+    name: string;
+    username: string;
+    email: string;
+  } | null;
+}
 
 export const App = () => {
+  const [userId, setUserId] = useState("0");
+  const [taskDescription, setTaskDescription] = useState("");
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [errorUser, setErrorUser] = useState(false);
+  const [todos, setTodos] = useState<Todo[] | []>([]);
+
+  useEffect(() => {
+    setTodos(tasks);
+  }, []);
+
+  let maxId = Math.max(...todos.map((o) => o.id)) + 1;
```

Comment:
The calculation of `maxId` should be updated after adding a new task. Consider moving this logic inside the `handleSubmit` function after updating the `todos` state.